installers/msi-language/Product.p.wxs

```diff
@@ -124,17 +124,23 @@
 			<Publish Dialog="ProgressDlg" Control="Cancel" Property="LAST_DIALOG" Value="ProgressDlg">1</Publish>
 			<Publish Dialog="WelcomeDlg" Control="Cancel" Property="LAST_DIALOG" Value="WelcomeDlg">1</Publish>
 			<Publish Dialog="MaintenanceWelcomeDlg" Control="Cancel" Property="LAST_DIALOG" Value="MaintenanceWelcomeDlg">1</Publish>
+			<Publish Dialog="MaintenanceTypeDlg" Control="Cancel" Property="LAST_DIALOG" Value="RemoveDlg">1</Publish>
 			<Publish Dialog="LicenseAgreementDlg" Control="Cancel" Property="LAST_DIALOG" Value="LicenseAgreementDlg">1</Publish>
 			<Publish Dialog="InstallDirDlg" Control="Cancel" Property="LAST_DIALOG" Value="InstallDirDlg">1</Publish>
-			<Publish Dialog="VerifyReadyDlg" Control="Cancel" Property="LAST_DIALOG" Value="VerifyReadyDlg">1</Publish>
+			<Publish Dialog="VerifyReadyDlg" Control="Cancel" Property="LAST_DIALOG" Value="VerifyReadyDlg">NOT Installed</Publish>
+			<Publish Dialog="VerifyReadyDlg" Control="Cancel" Property="LAST_DIALOG" Value="VerifyRemoveReadyDlg">WixUI_InstallMode = "Remove"</Publish>
 
 			<Publish Dialog="PrivacyConsentDlg" Control="Next" Property="LAST_DIALOG" Value="Post_PrivacyConsentDlg">1</Publish>
 			<Publish Dialog="ProgressDlg" Control="Next" Property="LAST_DIALOG" Value="Post_ProgressDlg">1</Publish>
 			<Publish Dialog="WelcomeDlg" Control="Next" Property="LAST_DIALOG" Value="Post_WelcomeDlg">1</Publish>
 			<Publish Dialog="MaintenanceWelcomeDlg" Control="Next" Property="LAST_DIALOG" Value="Post_MaintenanceWelcomeDlg">1</Publish>
+			<Publish Dialog="MaintenanceTypeDlg" Control="RemoveButton" Property="LAST_DIALOG" Value="Post_RemoveDlg">1</Publish>
 			<Publish Dialog="LicenseAgreementDlg" Control="Next" Property="LAST_DIALOG" Value="Post_LicenseAgreementDlg">1</Publish>
 			<Publish Dialog="InstallDirDlg" Control="Next" Property="LAST_DIALOG" Value="Post_InstallDirDlg">1</Publish>
-			<Publish Dialog="VerifyReadyDlg" Control="Install" Property="LAST_DIALOG" Value="Post_VerifyReadyDlg">1</Publish>
+			<Publish Dialog="VerifyReadyDlg" Control="Install" Property="LAST_DIALOG" Value="Post_VerifyReadyDlg">NOT Installed</Publish>
+			<Publish Dialog="VerifyReadyDlg" Control="InstallNoShield" Property="LAST_DIALOG" Value="Post_VerifyReadyDlg">NOT Installed</Publish>
```

Comment:
I added this line, and the line for `RemoveNoShield` below, because that is the Control's name in the `VerifyReadyDlg` if no UAC is necessary ie., if the user is already an administrator.  So, we may not have correctly caught these cases before...

Cf., https://github.com/wixtoolset/wix3/blob/2ed6d7906c87bb65b75cfa0020e38a1b1dffa49b/src/ext/UIExtension/wixlib/VerifyReadyDlg.wxs#L19